src/platform/user/authentication/datadog/utilities.js

```diff
@@ -0,0 +1,104 @@
+import { datadogLogs } from '@datadog/browser-logs';
+
+const DATA_DOG_TOKEN = 'pube9cdb0054ba0126c0adac86ff8ac50e2';
+const DATA_DOG_SERVICE = 'benefits-notice-of-disagreement';
```

Comment:
Can we change the service name to something `octo-identity` or something similar?